module-1/assignments/activity-1.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>​
+<myMessage>
+    <myMessage>​
```

Comment:
`myMessage` does not make sense in this case. should have used different name for the element

Reply:
and you have the same name for the root and nested element. not a good practise either